huaweicloud/services/evs/resource_huaweicloud_evs_volume.go

```diff
@@ -149,6 +153,19 @@ func ResourceEvsVolume() *schema.Resource {
 	}
 }
 
+func buildBssParamParams(d *schema.ResourceData) *cloudvolumes.BssParam {
+	bp := cloudvolumes.BssParam{
+		ChargingMode: d.Get("charging_mode").(string),
+	}
+	if strings.EqualFold(bp.ChargingMode, "prePaid") {
```

Comment:
the caller has checked this condition, can we skip it?

Reply:
yeah, that is a useless check.